lib/b2b/rbac_local.ts

```diff
@@ -0,0 +1,80 @@
+import { Policy, RBAC } from "./rbac";
+import { fetchConfig } from "../shared";
+import { ClientError } from "../shared/errors";
+
+// We want to refresh if the policy is more than 10 minutes old
+const MAX_AGE_MS = 1000 * 60 * 10;
```

Comment:
Could we potentially make this shorter? I was thinking that since JWTs expire every 5 minutes, we might want to refresh this more often as well